src/igvfd/types/institutional_certificate.py

```diff
@@ -28,3 +28,18 @@ class InstitutionalCertificate(Item):
 
     set_status_up = []
     set_status_down = []
+
+    @calculated_property(schema={
+        'title': 'Data Use Limitation Summary',
+        'type': 'string',
+        'description': 'A unique identifier for the ontology term, reformatted from the original term ID.',
```

Comment:
Description sounds like another property, probably just a copy paste mistake?

Reply:
Yeah, I didn't even notice the description.